Mu2eG4/inc/InitEnvToolBase.hh

```diff
@@ -1,28 +1,21 @@
 #ifndef Mu2eG4_InitEnvToolBase_hh
 #define Mu2eG4_InitEnvToolBase_hh
 
-#include "art/Framework/Services/Registry/ServiceHandle.h"
-#include "art_root_io/TFileService.h"
-
-#include "Offline/Mu2eG4Helper/inc/VolumeInfo.hh"
-#include "Offline/ConfigTools/inc/SimpleConfig.hh"
-
 namespace mu2e {
 
+  class VolumeInfo;
+  class SimpleConfig;
+
   class InitEnvToolBase {
-  protected:
-    std::string _name;
   public:
 
     InitEnvToolBase() noexcept = default ;
```

Comment:
We should make it impossible to construct an object of base type.  I believe we can do this by making the default c'tor protected.

I see that you removed the _name functionality in it's entirety.